src/test/regress/sql/multi_utility_statements.sql

```diff
@@ -101,7 +101,7 @@ COPY (
 -- check copying to file
 -- (quiet off to force number of copied records to be displayed)
 \set QUIET off
-COPY nation TO '/dev/null';
+\COPY nation TO :dev_null;
```

Comment:
When I run the tests, a file named `:dev_null` is created in the current directory. Maybe this variable is not passed to all test scripts.

Reply:
Addressed by 9436138